governance/votes/Executive vote - June 4, 2021.md

```diff
@@ -0,0 +1,96 @@
+- **80,000 DAI** will be distributed to the **[GovAlpha (GOV-001)](https://mips.makerdao.com/mips/details/MIP39c2SP3)** core unit at **0x01D26f8c5cC009868A4BF66E268c17B057fF7A73**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP3) and modified within this forum [post](https://forum.makerdao.com/t/govalpha-administered-multi-sig/8304/3).
+- **510,000 DAI** will be distributed to **[Protocol Engineering (PE-001)]()** core unit at **0xe2c16c308b843eD02B09156388Cb240cEd58C01c** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP7)
+- **126,117 DAI** will be distributed to the **[Growth (GRO-001)](https://mips.makerdao.com/mips/details/MIP39c2SP4)**  core unit at **0x7800C137A645c07132886539217ce192b9F0528e**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP4).
+- **44,375 DAI** will be distributed to the **[Content Production (MKT-001)](https://mips.makerdao.com/mips/details/MIP39c2SP5)**  core unit at **0x6A0Ce7dBb43Fe537E3Fd0Be12dc1882393895237** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP41c4SP5) and modified within this forum [post](https://forum.makerdao.com/t/adding-content-production-multisig/8428/5).
+- **642,135 DAI** will be distributed to **Sustainable Ecosystem Scaling (SES-001)** core unit auditors wallet at **0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP10).
+
+### Reduce the PSM-USDC-A Fee Out (`tout`)
+
+Due to the positive outcome of [this](https://vote.makerdao.com/polling/Qmc7PYCn?network=mainnet#poll-detail) recent governance poll. The PSM-USDC-A Fee Out (`tout`) parameter will be decreased from 0.04% to 0% if this executive proposal passes.
+
+### Liquidations 2.0 Migrations
+
+To resolve technical debt and ensure that the Maker Protocol remains as maintainable as possible the Protocol Engineering Core Unit have proposed moving the last remaining stablecoin collateral types to the Liquidations 2.0 framework, as detailed in this forum [post](https://forum.makerdao.com/t/architectural-debt-cleanup-migrating-remaining-collaterals-to-liquidations-2-0/8515). Liquidations will be disabled for all of the following collateral types, and they will have placeholder auction parameters in the Liquidations 2.0 system if this executive proposal passes.
+- USDC-A
+- USDC-B
+- TUSD-A
+- USDT-A
+- PAXUSD-A
+- GUSD-A
+- PSM-USDC-A
+- UNIV2DAIUSDC-A
+
+### Increase Global Debt Ceiling
+
+Due to the recent issue in which the Global Debt Ceiling limited usage of the Maker Protocol for a number of days, the Protocol Engineering Core Unit has proposed increasing the Global Debt Ceiling in this forum [post](https://forum.makerdao.com/t/propose-to-raise-the-global-debt-ceiling/8516). The Global Debt Ceiling will be increased by 500 million DAI if this executive proposal passes.
```

Comment:
I know we usually have a "from" to "to" format for these increases that wouldn't work with the Global DC

Reply:
Would it be worth saying something like:

Typically, the Global Debt Ceiling is the sum of all available Debt Ceilings. To alleviate the recent issue, this measure proposes adding a fixed amount of 500M to the sum of the debt ceilings, effectively raising the Global Debt Ceiling by 500M DAI.